### ISSUE TITLE
CEL spec is too protobuf centric?

### ISSUE THREAD

**Issue by jpbetz** [google/cel-spec#217]
We're using CEL in Kubernetes to integrate with OpenAPIv3 schema types.  When providing developers with the CEL spec as reference, it's a bit difficult to explain how our "object with fields" type maps to CEL types, because the spec doesn't have a term for this type that is independent of protobuf. For now we say in our documentation that our "object with fields" type maps to "message", but this gets a bit confusing.

**Comment by TristonianJones** [comment]
Internally we've started shifting toward referring to protos as a specialized `Struct`, since it's a more generic term for an object with typed fields. Does this line up with your thinking as well?

**Comment by jpbetz** [comment]
> Internally we've started shifting toward referring to protos as a specialized `Struct`, since it's a more generic term for an object with typed fields. Does this line up with your thinking as well?

Yes, "struct" would work well. It's probably my first choice. "object" is a bit of a problem for OpenAPI because it refers generically to all kinds of objects with fields (I.e "maps" and "structs").
